src/pybind/fst/fst_pybind.h

```diff
@@ -21,6 +21,176 @@
 
 #include "pybind/kaldi_pybind.h"
 
+#include "fst/fst.h"
+
+template <typename A>
+void pybind_state_iterator_base_impl(py::module& m,
```

Comment:
just curious where all this kind of thing is needed.  Doing things like state iteration in Python will be very slow.

Reply:
I just wrapped it for completeness.

Do we need to iterate an FST in Python? If not, I can remove them.

Reply:
imho, it can be useful sometimes in scripts.

Reply:
So we can just keep it and let the user choose whether to use it or not?